src/zenml/integrations/deepchecks/materializers/deepchecks_results_materializer.py

```diff
@@ -76,6 +78,24 @@ def save(self, result: Union[CheckResult, SuiteResult]) -> None:
         serialized_json = result.to_json(True)
         io_utils.write_file_contents_as_string(filepath, serialized_json)
 
+    def save_visualizations(
+        self, result: Union[CheckResult, SuiteResult]
+    ) -> Dict[str, VisualizationType]:
+        """Saves visualizations for the given Deepchecks result.
+
+        Args:
+            result: The Deepchecks result to save visualizations for.
+
+        Returns:
+            A dictionary of visualization URIs and their types.
+        """
+        visualizations = super().save_visualizations(result)
+        visualization_path = os.path.join(self.uri, HTML_FILENAME)
+        with fileio.open(visualization_path, "w") as f:
+            result.save_as_html(f)
+        visualizations[visualization_path] = VisualizationType.HTML
+        return visualizations
+
```

Comment:
> Is that part of a backwards-compatibility strategy that involves deprecation now and removal in a later release ? Then maybe throw a warning in the legacy visualizer code to inform people that it will go away.

I'm deprecating ALL visualizers with this, so the corresponding deprecation warning is logged in the base visualizer not here. Don't you think we need to keep the current visualizers at least for a sprint or two? I'd also love to delete them right away actually, if you agree I'll dispose of them directly :)

Reply:
I say nuke 'em !

Reply:
From orbit

Reply:
It's the only way to be sure

Reply:
let's do it 🚀